lib/ts/recipe/session/index.ts

```diff
@@ -82,7 +82,7 @@ export default class SessionWrapper {
     }
 
     static createJWT(payload?: any, validitySeconds?: number) {
-        let jwtRecipe = Recipe.getInstanceOrThrowError().jwtRecipe;
+        let jwtRecipe = Recipe.getInstanceOrThrowError().openIdRecipe?.jwtRecipe;
```

Comment:
openId Recipe should have all the function exposed by jwt recipe. This means, you don't need a ref to the jwt recipe at all in this file.

Reply:
Done